# this file is used to configure the behavior
# of the linear controller node


# parameters for the linear controller
controller:
  # proportionalities
  K_v: 0.5
  K_alpha: 2.0
  K_beta: -1.0
  # limits
  v_max: 0.25 # [m/s]
  omega_max: 0.8 # [rad/s]
  # goal threshold (setponit-state distance norm)
  # unitless, since it normalizes (x,y,theta) [m,m,rad]
  threshold: 0.15
  ignore_yaw: false
